src/test/ij.c

```diff
@@ -30,6 +30,8 @@
 #include "superlu_ddefs.h"
 #endif
 
+#include "../utilities/hypre_cuda_utils.h"
```

Comment:
Users should not include files from the source tree, only installed files.  We try to make the test drivers somewhat independent of internal hypre stuff.  Why is this needed here?

Reply:
There are a few places in this files directly calling CUDA routines. I included this file to simply include the needed headers.